sdk/tables/azure-data-tables/inc/azure/data/tables/account_sas_builder.hpp

```diff
@@ -114,13 +114,13 @@ namespace Azure { namespace Data { namespace Tables { namespace Sas {
     All = ~0,
   };
 
-  inline AccountSasServices operator|(AccountSasServices lhs, AccountSasServices rhs)
+  inline AccountSasServices operator|(AccountSasServices const lhs, AccountSasServices const rhs)

```

Comment:
For each type that defines logical operators, could you please define a 7 `constexpr` operators for `^`, `&`, `|`, `~`, `^=`, `&=`, and `|=`, and also have a single unit test per type for each of 7 operations?

Reply:
BTW, if you are taking by value, `const` is not needed.

Reply:
done